### DISCUSSION TITLE
libncurses.so.5 Error

### DISCUSSION THREAD

**Discussion by azim7091** [iraf-community/iraf#140] [Q&A] [answered]
Hi
I am trying to install IRAF on fedora29, but after installation by the help of this link (https://www.astr.tohoku.ac.jp/~akhlaghi/irafinstall.html), when I enter cl in the terminal, I get this error:

/iraf/iraf/bin.linux64/vocl.e: error while loading shared libraries: libncurses.so.5: cannot open shared object file: No such file or directory

I have tried different things without any answer. I would be very grateful for any help.

Thanks
Azim

**Comment by olebole** [comment, marked as answer]
The reason is probably that you didn't install the 32 bit libraries. However, the installation procedure shown in your link are for an old version of IRAF, which is no longer supported.
I would instead recommend to switch to the latest sources and compile it yourself -- see [these instructions](https://iraf-community.github.io/install) for the procedure. Please open another issue if you encounter any problems with these.
